### PR TITLE
Open Energy Dashboard Portfolio

### PR DESCRIPTION
**Pull request by Rakesh-Ranga-Buram**

### CODE REVIEW

File: docs/portfolio.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ We are especially interested in projects supporting researchers, focused on rese
   
 **[Where's Religion Mobile](project_wheres_religion_mobile/about.md)** - This Project is to create an application for ethnographers to capture the events happening around them along with some audio/visual representation of the event.  
 
+**[Open Energy Dashboard](project_open_energy_dashboard/about.md)** - Open Energy Dashboard is a user-friendly way to display energy information from smart energy meters or uploaded via CSV files.
```
Comment (kungfuchicken) [inline docs/portfolio.md:48]: Please move this to the External Projects section that @MutyaSowmya123 created. It's merged into main, so you can either merge main into your branch or rebase your branch off of main to get this section.

File: docs/project_open_energy_dashboard/about.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+id: about
+title: Open Energy Dashboard
+custom_edit_url: null
+---
+![Alt](OED.png) 
+
+## Overview
+
+The Open Energy Dashboard (OED) is a crucial open-source project that facilitates the visualization of energy information through a web-based application. Its significance lies in promoting collaboration and knowledge-sharing to advance common sustainability goals. Unlike other proprietary dashboards, OED is free, portable, and driven by user needs, embodying principles such as perpetual freedom, shared advancements, and user-centric design. The typical user, ranging from energy managers to sustainability professionals, benefits from its user-friendly interface and diverse functionalities, allowing easy management of energy data without the need for external support.
+
```
Comment (kungfuchicken) [inline docs/project_open_energy_dashboard/about.md:11]: Please add as much detail in the ### Information section as you can. It doesn't need to have everything, but there are things you know (repo URL, client, tech lead, adoption date, technologies used, type, license).

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
you're making progress.

**Comment by Rakesh-Ranga-Buram** [comment]
Added the requested changes. @kungfuchicken can you please review this PR?

**Review by kungfuchicken — APPROVED** [review]
good enough for now.

**Pull request merged**